unified-codes/apps/web/src/components/containers/explorer/ExplorerTableRows.tsx

```diff
@@ -24,7 +24,7 @@ const styles = (theme: ITheme) => {
     cursor: 'pointer',
     '&:hover': { backgroundColor: theme.palette.background.toolbar },
     '& td:last-child': { borderRight: 0 },
-    '& td:first-child': { fontWeight: 700 },
+    '& td:first-child': { fontWeight: 700, minWidth: 125 },
```

Comment:
the table could do with a `minWidth` too, when squished, the type column disappears and the top menu is also lost.

![Screen Shot 2021-08-16 at 12 28 26 PM](https://user-images.githubusercontent.com/9192912/129497891-876a5a0f-f2a1-49dc-b335-bf0361cdf074.png)